connector-sdk/document/src/main/java/io/camunda/document/store/InMemoryDocumentStore.java

```diff
@@ -97,4 +97,8 @@ public void deleteDocument(CamundaDocumentReference reference) {
   public void clear() {
     documents.clear();
   }
+
```

Comment:
found it helpful for test purposes, and this is the InMemory store not a production one, I guess it's ok